recipes/eliot-tree/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "eliot-tree" %}
+{% set version = "18.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d492154cbc4093faf4492dbf503e87f51e27f89c7fc8697201157e0fb8d7d027
+
+build:
+  number: 0
+  skip: true  # [py<35 or win]
```

This package can probably be `noarch: python` and thus you should be able to remove the `skip` here
```suggestion
  noarch: python
```

---

My experience with `noarch` is that it doesn't work when you create scripts (as this package does) because the scripts for Linux/macOS and Windows are different. Is that not the case?

---

Yes, in this case you must explicitly define the entry points https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

---

Or I can just leave it as is?

---

OK, done.